config/index.js

```diff
@@ -399,7 +399,7 @@ var conf = convict({
     enabled: {
       doc: 'enable signin confirmation',
       default: false,
-      env: 'SIGNIN_CONFIRMATION_ENABLE'
+      env: 'SIGNIN_CONFIRMATION_ENABLED'
```

Comment:
This one, I renamed for consistency with other feature toggle-flags that use "ENABLED" rather than "ENABLE"